pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -131,6 +131,29 @@ def __call__(self, value, system):
             del response.headers['Connection']
         return content
 
+    @staticmethod
+    def get_wms_url_params():
+        """
+        Returns the list of additionally configured wms_url_params.
+
+        :return: The configured wms_url_params.
+        :rtype: list
+        """
+        result = {}
+        wms_url_params = Config.get('print', {}).get('wms_url_params', False)
+        if wms_url_params:
+            log.debug("get_wms_url_params() read configuration {}".format(wms_url_params))
+            if isinstance(wms_url_params, dict):
+                result = wms_url_params
+            else:
+                log.warning("get_wms_url_params() ignoring unaccepted configuration value {}"
```

Comment:
Shoudln't it still return the default values in this case?

Reply:
IMO in that case it is better to return nothing, this way it is more likely that the user (the bad configurator) will notice something is wrong.

Reply:
Ok for me